Services/Repository/classes/class.ilRepUtil.php

```diff
@@ -141,6 +141,7 @@ static public function deleteObjects($a_cur_ref_id, $a_ids)
 			// SAVE SUBTREE AND DELETE SUBTREE FROM TREE
 			$affected_ids = array();
 			$affected_parents = array();
+			$org_unit_ids = array();
```

Comment:
@bseglias could you remove the changes from ilRepUtil and ilRepUtilGUI and override `confirmedDeleteObject` from `ilObjectGUI` in `ilObjOrgUnitGUI` to just remove the orgus directly by using `$ru->removeObjectsFromSystem($_GET["ref_id"], ilSession::get("saved_post"));`? 

`ilRepUtil::showDeleteConfirmation()` then need something like an additional parameter `$additional_message` defaulted to null.